test/lib/query.spec.js

```diff
@@ -293,4 +294,89 @@ describe('Pimp My Sql :: Query', function() {
     })
   })
 
+  describe('::getTimestamp', () => {
+    it('should return a proper date and time for a unix timestamp', () => {
+
+      const input = 1515188138
```

Comment:
What happens if a millisecond based timestamp is given?

Reply:
They you smack that developer for using milliseconds as a timestamp. The industry standard is seconds.